pkg/init/backend/interactive.go

```diff
@@ -90,11 +90,22 @@ loop:
 	return result, nil
 }
 
+type StarterProjectSorter []v1alpha2.StarterProject
+
+func (starterProject StarterProjectSorter) Len() int { return len(starterProject) }
+func (starterProject StarterProjectSorter) Swap(i, j int) {
+	starterProject[i], starterProject[j] = starterProject[j], starterProject[i]
+}
+func (starterProject StarterProjectSorter) Less(i, j int) bool {
+	return starterProject[i].Name < starterProject[j].Name
+}
+
 func (o *InteractiveBackend) SelectStarterProject(devfile parser.DevfileObj, flags map[string]string) (*v1alpha2.StarterProject, error) {
 	starterProjects, err := devfile.Data.GetStarterProjects(parsercommon.DevfileOptions{})
 	if err != nil {
 		return nil, err
 	}
+	sort.Sort(StarterProjectSorter(starterProjects))
```

Comment:
You could use the simpler function `sort.Slice`, which needs only the equivalent of the `Less` method